app/views/payments/openPayments.scala.html

```diff
@@ -72,15 +76,16 @@ <h1 class="heading-xlarge">@messages("openPayments.heading")</h1>
 
     <div id="payments-information">
       <p id="processing-time">@Messages("openPayments.processingTime")</p>
-      <span id="direct-debits" class="heading-small">Direct debits</span>
     </div>
 
-    <div id="check-direct-debit" class="form-group">
-      <p>
-        @Messages("openPayments.directDebitLink") <a href="#">@Messages("openPayments.directDebitLinkText")
-      </a> @Messages("openPayments.directDebitNotSure")</p>
-
-    </div>
+    @if(appConfig.features.allowPayments()) {
```

Comment:
The allowPayments feature is seperate to the DD feature so it shouldn't stop users from going to DD